index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+ <meta charset="UTF-8">
+ <meta http-equiv="X-UA-Compatible" content="IE=edge">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <title>Document</title>
+ <link rel="stylesheet" href="./index.css">
+ <link rel="stylesheet" href="./navbar.css">
+</head>
+<body>
+<div id="navbar" >
+  <div id="menu">
+   <a href="#projects" id="profile-link"><p>Projects</p></a>
+  </div>
+  <div id="github">
+   <a href="https://github.com/IsmaelMastronardi" target="_blank"id="profile-link"><p>Github</p></a>
+  </div>
+</div>
```

Comment:
- [ ] Great effort making use of inline comment, but as part of this milestone I would kindly suggest you give the reason `WHY` you are requesting for a review.
- [ ] Also I would suggest you make your review clear and easy to understand by the student.

Review guideline :point_down: 
1. The student always includes an explanation, a "why", for each requested change. For example:
      - This is a well-explained request: Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible
      - This request lacks a "why": Why don't you use an article tag here instead of a div?

2. The student comments are clear and easy to understand. For Example:
      - This is a clear comment: The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?
      - This is a vague comment: The README is not descriptive, why don't you make it better?

> N.B: kindly edit the review and make the changes, you don't need to create a new inline comment, just simply edit the ones you have given before and make it look perfect before you resubmit for review. Kindly add it to all the comments you have initially.
Thank you.